Sources/BeeTracking/OISTBeeVideo+Batches.swift

```diff
@@ -62,6 +62,84 @@ extension OISTBeeVideo {
     return statistics.normalized(stacked)
   }
 
+  /// Returns a batch of locations of background
+  /// bee bounding boxes.
+  public func makeForegroundBoundingBoxes(
+    patchSize: (Int, Int),
+    batchSize: Int = 200
+  ) -> [(frame: Tensor<Double>?, obb: OrientedBoundingBox)] {
+    /// Anything not completely overlapping labels
+    var deterministicEntropy = ARC4RandomNumberGenerator(seed: 42)
+    let frames = self.randomFrames(self.frames.count, using: &deterministicEntropy)
+
+    // We need `batchSize / frames.count` patches from each frame, plus the remainder of the
+    // integer division.
+    var patchesPerFrame = Array(repeating: batchSize / frames.count, count: frames.count)
+    patchesPerFrame[0] += batchSize % frames.count
+
+    let obbs = zip(patchesPerFrame, frames).flatMap { args -> [(frame: Tensor<Double>?, obb: OrientedBoundingBox)] in
+      let (patchCount, (_, (frame, labels))) = args
+      let locations = labels.randomSelectionWithoutReplacement(k: patchCount, using: &deterministicEntropy).map(\.location.center)
+      return locations.map { location -> (frame: Tensor<Double>?, obb: OrientedBoundingBox) in
+        return (frame: frame, obb: OrientedBoundingBox(
+            center: location,
+            rows: patchSize.0, cols: patchSize.1))
+      }
+    }
+
+    return obbs
+  }
+  
+  /// Returns a batch of locations of background
```

Comment:
I thought we already had this type of functionality. Where is the old code, and how is it different? No copy/pasta please!